src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+import ui.Ui;
+
+public class Duke {
+
```

Comment:
Perhaps the work in the Ui and Duke can be redistributed?  As of now Duke.java is just the launcher for the real Duke program?